internal/connect/command_offset_describe.go

```diff
@@ -0,0 +1,175 @@
+	Metadata connectv1.ConnectV1ConnectorOffsetsMetadata `json:"metadata" yaml:"metadata"`
+}
+
+func (c *offsetCommand) newDescribeCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:               "describe <id>",
+		Short:             "Describe connector offsets.",
+		Args:              cobra.ExactArgs(1),
+		RunE:              c.describe,
+		ValidArgsFunction: pcmd.NewValidArgsFunction(c.validArgs),
+		Example: examples.BuildExampleString(
+			examples.Example{
+				Text: "Describe offsets for a connector in the current or specified Kafka cluster context.",
+				Code: "confluent connect offset describe lcc-123456",
+			},
+			examples.Example{
+				Code: "confluent connect offset describe lcc-123456 --cluster lkc-123456",
+			},
+		),
+	}
+
+	cmd.Flags().Int32("staleness-threshold", 120, "Repeatedly fetches offsets, until we get an offset with an observed time within staleness threshold in seconds. (min 5)")
+	cmd.Flags().Int32("refetch-timeout", 30, "Max time in seconds to wait until we get an offset within the staleness threshold.")
+
+	pcmd.AddClusterFlag(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddContextFlag(cmd, c.CLICommand)
+	pcmd.AddEnvironmentFlag(cmd, c.AuthenticatedCLICommand)
+	pcmd.AddOutputFlag(cmd)
+
+	return cmd
+}
+
+func (c *offsetCommand) describe(cmd *cobra.Command, args []string) error {
+	kafkaCluster, err := kafka.GetClusterForCommand(c.V2Client, c.Context)
+	if err != nil {
+		return err
+	}
+
+	environmentId, err := c.Context.EnvironmentId()
+	if err != nil {
+		return err
+	}
+
+	connector, err := c.V2Client.GetConnectorExpansionById(args[0], environmentId, kafkaCluster.ID)
+	if err != nil {
+		return err
+	}
+
+	stalenessThreshold, err := cmd.Flags().GetInt32("staleness-threshold")
+	if err != nil {
+		return err
+	}
+
+	if stalenessThreshold < 5 {
+		return fmt.Errorf("staleness-threshold cannot be less than 5 seconds")
+	}
+
+	refetchTimeout, err := cmd.Flags().GetInt32("refetch-timeout")
+	if err != nil {
+		return err
+	}
+
+	connectorName := connector.Info.GetName()
+
+	var apiErr error
+	var offsets connectv1.ConnectV1ConnectorOffsets
+
+	_ = retry.Retry(time.Second, time.Duration(refetchTimeout)*time.Second, func() error {
+		offsets, apiErr := c.V2Client.GetConnectorOffset(connector.Info.GetName(), environmentId, kafkaCluster.ID)
+		if apiErr != nil {
+			return apiErr
+		}
+
+		if offsets.HasMetadata() && offsets.Metadata.HasObservedAt() && int32(time.Since(*offsets.Metadata.ObservedAt).Seconds()) <= stalenessThreshold {
+			return nil
+		}
+		return fmt.Errorf("got stale offsets, fetching again")
+	})
+
+	if apiErr != nil {
+		return apiErr
+	}
+
+	if output.GetFormat(cmd) == output.Human {
+		return printHumanDescribeOffset(cmd, offsets, args[0], connectorName)
+	}
+
+	return printSerializedDescribeOffsets(cmd, offsets, args[0], connectorName)
+}
+
+func printHumanDescribeOffset(cmd *cobra.Command, offsets connectv1.ConnectV1ConnectorOffsets, id, name string) error {
+	var offsetInfo []map[string]any
```

Comment:
Looks like this should be declared before the second block since it's used there

Reply:
Moved